### PR TITLE
Chromosome type DOS-DP

### PR DESCRIPTION
**Pull request by sabrinatoro**
Addresses #4281
I also:
- cleaned up ring_chromosome_anomaly pattern
- updated ring chromosome disorder name

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@sabrinatoro the test is failing

**Comment by sabrinatoro** [comment]
@nicolevasilevsky I know! :-( And I am checking the yaml with the validator... I don't know why. 
I have made more tweaks... we'll see...

**Comment by nicolevasilevsky** [comment]
this happens to be almost every time I create a new pattern!

**Comment by sabrinatoro** [comment]
I need help. 
I "think" the DOS-DP templates I made/updated are ok (the logs do not report anything about them anymore). Therefore I have no idea what is the errors are at this point. Can you help, please and thank you? @nicolevasilevsky @matentzn

**Comment by nicolevasilevsky** [comment]
@sabrinatoro 
Not sure why, but if you format the logical defs like this, it will work (I just did a test and it passed - #4312)

equivalentTo:
  text: '''ring chromosome disorder'' and ''disease arises from structure'' some %s'
  vars:
  - chromosome

File: src/patterns/dosdp-patterns/chromosome_type.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  vars:
+  - chromosome
+
+def:
+  text: A chromosomal disorder in which %s is affected.
+  vars:
+  - chromosome
+
+equivalentTo:
+  text: "'chromosomal disorder' and '('disease arises from structure' some %s)'"
```
Comment (matentzn) [inline src/patterns/dosdp-patterns/chromosome_type.yaml:37]: Redundant `'` around the `()` expression, i.e.

```
  text: "'chromosomal disorder' and '('disease arises from structure' some %s)'"
```

should be

```
  text: "'chromosomal disorder' and ('disease arises from structure' some %s)"
```

File: src/patterns/dosdp-patterns/ring_chromosome_anomaly.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  vars:
+  - chromosome
+
+def:
+  text: A chromosomal disorder consisting of the presence of ring chromosome %s.
+  vars:
+  - chromosome
+
+equivalentTo:
+  text: "'ring chromosome disorder' and '('disease arises from structure' some %s)'"
```
Comment (matentzn) [inline src/patterns/dosdp-patterns/ring_chromosome_anomaly.yaml:37]: Same as above

**Review by matentzn — CHANGES_REQUESTED** [review]

**Comment by sabrinatoro** [comment]
Thank you @matentzn 
@nicolevasilevsky , it is ready for your revision! Thanks

**Review by nicolevasilevsky — COMMENTED** [review]
did you intend to delete 4 files?

**Comment by sabrinatoro** [comment]
@nicolevasilevsky 
yes, I deleted the files which didn't have the ".yaml" extension. All these files have a '.yaml' version.

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by nicolevasilevsky** [comment]
Looks good to me!

**Pull request merged**